src/main/steps/respondent/hub-page/content.ts

```diff
@@ -108,13 +108,26 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
     line1: `Your ${partner}’s application for a 'conditional order' has been accepted. The court agrees that you are entitled to ${
       isDivorce ? 'get divorced' : 'end your civil partnership'
     }.`,
+    line2: `A judge will 'pronounce' (read out) your conditional order at a hearing. The hearing will take place at ${
+      userCase.coCourt === ConditionalOrderCourt.BIRMINGHAM ? birmingham : buryStEdmunds
+    } on ${dayjs(userCase.coDateAndTimeOfHearing).format('D MMMM YYYY')} at ${dayjs(
+      userCase.coDateAndTimeOfHearing
+    ).format('h:mmA')}.`,
+    line3: `You do not need to come to the hearing, unless you want to object. You must contact the court by ${dayjs(
+      userCase.coDateAndTimeOfHearing
+    )
+      .subtract(7, 'day')
+      .format('D MMMM YYYY')} if you want to attend.`,
     line4: `After your conditional order has been pronounced, your ${partner} will then be able to apply for a 'final order' on ${dayjs(
       userCase.coDateAndTimeOfHearing
     )
       .add(43, 'day')
       .format('D MMMM YYYY')}. This is the final step in the ${
       isDivorce ? 'divorce ' : ''
     }process and will legally end your ${isDivorce ? 'marriage' : 'civil partnership'}.`,
+    line5: `You can <a class="govuk-link" href="/downloads/certificate-of-entitlement" download="Certificate-of-entitlement">view and download your ‘certificate of entitlement for a conditional order’</a>. This is the document that says the court does not see any reason why you cannot ${
+      isDivorce ? 'get divorced' : 'end your civil partnership'
+    }.`,
```

Comment:
shall we start moving all the links into the template from now on?

Reply:
sure